README.md

```diff
@@ -19,7 +19,13 @@ To build and serve the site, you'll need these tools:
 
 ### Setup
 
-Once you've installed the [prerequisites](#prerequisites), get local packages:
+Install the Docsy submodule with the following command:
```

Comment:
How about:

> Get the Docsy theme using the following command:

Reply:
Added! That's much better. Thank you, @chalin!